packages/formation/js/index.js

```diff
@@ -1,10 +1,13 @@
 import loadAccordionHandler from './accordion';
+import domready from 'domready';
 
 if (
   document.readyState === 'complete' &&
   (document.readyState !== 'loading' && !document.documentElement.doScroll)
 ) {
   loadAccordionHandler();
 } else {
-  document.addEventListener('DOMContentLoaded', loadAccordionHandler);
+  domready(() => {
```

Comment:
added this because I had issues when using this on vets-website. I saw this is what  uswds uses, so I tried it out. So far it works.

Reply:
can probably just call `domready(loadAccordionHandler)`

Reply:
Yeah your right